source/routing/raptor_api.cpp

```diff
@@ -305,48 +305,53 @@ static void add_pathes(EnhancedResponse& enhanced_response,
             path.items.back().stop_points.back()->is_zonal;
 
         if (journey_begin_with_address_odt) {
-            // nor crow fly section, but we'll have to update the start of the journey
+            // no crow fly section, but we'll have to update the start of the journey
             //first is zonal ODT, we do nothing, there is no walking
         }
-        else if (!path.items.front().stop_points.empty()
-                && use_crow_fly(origin, path.items.front().stop_points.front(), d)){
-            const auto sp_dest = path.items.front().stop_points.front();
-            type::EntryPoint destination_tmp(type::Type_e::StopPoint, sp_dest->uri);
-            bt::time_period action_period(path.items.front().departures.front(),
-                                          path.items.front().departures.front()+bt::minutes(1));
+        else if (!path.items.empty() && !path.items.front().stop_points.empty()){
```

Comment:
better `} else if (!path.items.empty() && !path.items.front().stop_points.empty()) {` (with a space between `)` and `{`, and `}` and `else` on the same line)